### ISSUE TITLE
VeSTige not loading VST

### ISSUE THREAD

**Issue by eugenelesnov** [LMMS/lmms#6237]
### Bug Summary

VeSTige not loading VST. When I choose my .dll nothing happens.

#### Steps to reproduce

1. Using Kubuntu 21.10 download last stable LMMS 1.2.2. Run .AppImage from terminal:

`./lmms-1.2.2-linux-x86_64.AppImage`

2. Add VeSTige into project then find .dll.
In my particular case here's [link](https://www.powerdrumkit.com/) for plugin.
 
#### Expected behavior

Expected to load VST plugin with VeSTige.

#### Actual behavior

Nothing happens on GUI. 

#### Affected LMMS versions

LMMS 1.2.2

#### Logs
<details>
  <summary>Click to expand</summary>
<pre>
```
./lmms-1.2.2-linux-x86_64.AppImage 
Carla does not appear to be installed.  That's OK, please ignore any related library errors.
Jack appears to be installed on this system, so we'll use it.
Cannot load library /tmp/.mount_lmms-1jJ3DLh/usr/lib/lmms/libcarlarack.so: (libcarla_native-plugin.so: невозможно открыть разделяемый объектный файл: Нет такого файла или каталога)
Cannot load library /tmp/.mount_lmms-1jJ3DLh/usr/lib/lmms/libcarlapatchbay.so: (libcarla_native-plugin.so: невозможно открыть разделяемый объектный файл: Нет такого файла или каталога)
Cannot load library /tmp/.mount_lmms-1jJ3DLh/usr/lib/lmms/libcarlabase.so: (libcarla_native-plugin.so: невозможно открыть разделяемый объектный файл: Нет такого файла или каталога)
Connection established.

Stream successfully created

0034:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
0034:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
0064:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
0064:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
006c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
006c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

002c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
002c:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
00c0:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
00c0:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
0024:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
0024:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

00c8:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
00c8:fixme:font:get_name_record_codepage encoding 20 not handled, platform 1.
Starting using X11Embed protocol.
QProcess: Destroyed while process ("/tmp/.mount_lmms-1jJ3DLh/usr/lib/lmms/RemoteVstPlugin") is still running.```
</pre>
</details>

**Comment by musikBear** [comment]
Do you have **WINE** installed, and was it installed before you created the AppImage?

**Comment by eugenelesnov** [comment]
@musikBear yes, I do. It's wine-6.0.2

**Comment by ahstro** [comment]
Which architecture are the VSTs built for? IIRC, 64bit VSTs don't run on LMMS in Linux yet, so I had to find 32bit versions of the ones I wanted. (I think this is fixed in 1.3.0, but no idea when that will reach release)

**Comment by eugenelesnov** [comment]
@ahstro My bad, I tried to load 64bit version of plugin...For sure, issue can be closed, I think. Thanks for feedback and have a nice day :)
